### PR TITLE
docs(cn): translate `RouterView slot` Chapter into Chinese

### PR DESCRIPTION
**Pull request by yionr**

### CODE REVIEW

File: packages/docs/.vitepress/translation-status.json
Comment (Jinjiang) [inline packages/docs/.vitepress/translation-status.json:1]: Same as https://github.com/vuejs/router/pull/2090#discussion_r1436745733

File: packages/docs/zh/guide/advanced/router-view-slot.md [hunk 1]
```diff
@@ -46,11 +44,11 @@ We can also use KeepAlive inside a Transition:
 </router-view>
 ```
 
-For more information about using RouterView with the Transition component, see the [Transitions](./transitions) guide.
+关于更多 RouterView 组件和 Transition 组件之间的互动，请参考 [Transitions](./transitions) 指南。
 
-## Passing props and slots
+## 传递 props 和 slots
```
Comment (Jinjiang) [inline packages/docs/zh/guide/advanced/router-view-slot.md:49]:
```suggestion
## 传递 props 和插槽
```

下同

File: packages/docs/zh/guide/advanced/router-view-slot.md [hunk 2]
```diff
@@ -60,16 +58,16 @@ We can use the slot to pass props or slots to the route component:
 </router-view>
 ```
 
-In practice, this usually isn't something you would want to do, as the route components would **all need to use the same props and slots**. See [Passing Props to Route Components](../essentials/passing-props) for other ways to pass props.
+实践中通常不会这么做，因为这样会导致所有路由组件 **都使用相同的 props 和 slots**。请查阅 [传递 props 给路由组件](../essentials/passing-props) 获取其他传递 props 的方式。
```
Comment (Jinjiang) [inline packages/docs/zh/guide/advanced/router-view-slot.md:61]:
```suggestion
实践中通常不会这么做，因为这样会导致所有路由组件**都使用相同的 props 和插槽**。请查阅[传递 props 给路由组件](../essentials/passing-props)获取其他传递 props 的方式。
```

File: packages/docs/zh/guide/advanced/router-view-slot.md [hunk 3]
```diff
@@ -60,16 +58,16 @@ We can use the slot to pass props or slots to the route component:
...
 
-## Template refs
+## 模板引用
 
-Using the slot allows us to put a [template ref](https://vuejs.org/guide/essentials/template-refs.html) directly on the route component:
+使用插槽可以让我们直接将 [模板引用](https://vuejs.org/guide/essentials/template-refs.html) 放置在路由组件上：
```
Comment (Jinjiang) [inline packages/docs/zh/guide/advanced/router-view-slot.md:65]:
```suggestion
使用插槽可以让我们直接将[模板引用](https://vuejs.org/guide/essentials/template-refs.html)放置在路由组件上：
```

**Review by Jinjiang — CHANGES_REQUESTED** [review]

File: packages/docs/zh/guide/advanced/router-view-slot.md [hunk 4]
```diff
@@ -46,11 +44,11 @@ We can also use KeepAlive inside a Transition:
...
 ```
 
-For more information about using RouterView with the Transition component, see the [Transitions](./transitions) guide.
+关于更多 RouterView 组件和 Transition 组件之间的互动，请参考 [Transitions](./transitions) 指南。
 
-## Passing props and slots
+## 传递 props 和插槽
 
-We can use the slot to pass props or slots to the route component:
+我们可以用插槽来传递 props 或 slots 给路由组件：
```
Comment (yionr) [inline packages/docs/zh/guide/advanced/router-view-slot.md:51]: After check , I find here yet another `slots`, should it translate? I have no idea.

Comment (Jinjiang) [reply packages/docs/zh/guide/advanced/router-view-slot.md:51]: Yes, I think so. No worries. Let me do this.

**Review by Jinjiang — APPROVED** [review]

**Pull request merged**